main/src/modules/bodySchema/sensoryProcessor/src/sensoryProcessor.cpp

```diff
@@ -282,10 +283,14 @@ bool SensoryProcessor::getMultimodalData()
     Size subPixWinSize(10,10), winSize(31,31);
 
     ImageOf<PixelRgb> *yarpImage = imgPortIn.read();
-    IplImage *cvImage = cvCreateImage(cvSize(yarpImage->width(), yarpImage->height()), IPL_DEPTH_8U, 3);
-    cvCvtColor((IplImage*)yarpImage->getIplImage(), cvImage, CV_RGB2BGR);
+    ImageOf<PixelBgr> tmp; tmp.resize(*yarpImage);
```

Comment:
Since there's yarp in the middle I was not 100% sure; it's not my code in the end and I deemed quite risky to perform optimization.

There was also a memory leak before: `cvImage` was not destroyed, if I'm not mistaken.